Src/Common/CorrelationIdLookupHelper.cs

```diff
@@ -119,22 +122,36 @@ public bool TryGetXComponentCorrelationId(string instrumentationKey, out string
                             {
                                 this.GenerateCorrelationIdAndAddToDictionary(instrumentationKey, appId.Result);
                             }
+#if !NETCORE
                             catch (Exception ex)
                             {
                                 CrossComponentCorrelationEventSource.Log.FetchAppIdFailed(this.GetExceptionDetailString(ex));
                             }
+#else
+                            catch (Exception)
```

Comment:
I haven't spent any amount of time in the .net core code base for AI. Perhaps @osvaldorosado can comment